static/js/tools/timeline_bulk_download.ts

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import axios from "axios";
+import { TimelineParams } from "./timeline_util";
+
+/**
+ * Downloads data for all places of a certain type for the specified ptpv's in
+ * the chart.
+ */
+function downloadBulkData(statVars, placeType) {
```

Comment:
add a param for ancestor place to make it more generic?

Reply:
done